docs/pages/x/api/date-pickers/time-picker.json

```
@@ -84,6 +84,10 @@
     "orientation": {
       "type": { "name": "enum", "description": "'landscape'<br>&#124;&nbsp;'portrait'" }
     },
+    "referenceDate": {
+      "type": { "name": "any" },
+      "default": "The closest valid date-time using the validation props, except callbacks such as `shouldDisableDate`."
```

The JSDoc is not great for time picker.
I can either:

- Remove the `such as ...` part to be coherent with all pickers
- Re-define the JSDoc on the time picker `shared.ts` file => not super coherent but allow to have the best output
- Accept that this JSDoc is still easy enough to understand
- Replace by `such as shouldDisableDate / shouldDisableTime` on all pickers

---

I would go with a variant of 1st option.
What do you think about this?
```suggestion
      "default": "The closest valid date-time using the validation props, except callbacks like `shouldDisable<...>`."
```